webapp-django/crashstats/topcrashers/views.py

```diff
@@ -104,8 +105,19 @@ def get_topcrashers_results(**kwargs):
                 if row['term'] in (1, -1):
                     hit['hang_count'] += row['count']
 
-            # Number of startup crashes.
-            hit['startup_percent'] = 0
+            # Number of crashes happening during startup. This is defined by
+            # the software, as opposed to the next methid which relies on
+            # the uptime of the software.
+            hit['startup_count'] = 0
+
+            sig_startup = hit['facets']['startup_crash']
+            for row in sig_startup:
+                if row['term'] in ('T', '1'):
+                    hit['startup_count'] = row['count']
+
+            # Is a startup crash if more than half of the crashes are happening
+            # in the first minute after launch.
+            hit['startup_crash'] = False
```

Comment:
Nit: Can you remove the blank line under this line. The comment now appears to belong only to this line, but it's not until you read into the for loop below that the comment makes sense. Perhaps it should be structured like this instead:
```python
# Assume it's not a startup crash
hit['startup_crash'] = False
# We consider it a startup crash if more than half of the crashes are happening
# in the first 5 minutes after launch. 
sig_startup = hit['facets']['histogram_uptime']
for row in sig_startup:
    ...
```